--- photos.py.2
+++ photos.py
@@ -190,9 +190,12 @@
                 folder_id = folder['recordName']
                 folder_obj_type = \
                     "CPLContainerRelationNotDeletedByAssetDate:%s" % folder_id
-                folder_name = base64.b64decode(
-                    folder['fields']['albumNameEnc']['value']).decode('utf-8')
-                query_filter = [{
+                try:
+                    folder_name = base64.b64decode(
+                        folder['fields']['albumNameEnc']['value']).decode('utf-8')
+                except KeyError:
+                    continue        
+                query_filter = [{
                     "fieldName": "parentId",
                     "comparator": "EQUALS",
                     "fieldValue": {
